src/cli/CMakeLists.txt

```diff
@@ -97,6 +115,14 @@ if(BUILD_TESTING)
     COMMAND $<TARGET_FILE:openstudio> labs -e "puts(OpenStudio::Model::Model.new())"
   )
 
+  add_test(NAME OpenStudioCLI.Labs.ruby_execute_line.embedded_gem
+    COMMAND $<TARGET_FILE:openstudio> labs -e "require 'minitest'"
+  )
+
+  add_test(NAME OpenStudioCLI.Labs.ruby_execute_line.embedded_native_gem
+    COMMAND $<TARGET_FILE:openstudio> labs -e "require 'oga'"
+  )
```

Comment:
New labs tests to check we can load: a "normal" embedded gem, and a "native extension" embedded gem.